### PR TITLE
Add highlighting to search results

### PR DESCRIPTION
**Pull request by tombye**
Add highlighting to show where matches occurred in each search result.

https://www.pivotaltracker.com/story/show/94070032

highlighting

### CODE REVIEW

File: app/assets/scss/_search-page.scss [hunk 1]
```diff
@@ -27,3 +27,10 @@
     border: 1px solid $border-colour;
   }
 }
+
+.search-result-excerpt {
+  em {
```
Comment (quis) [inline app/assets/scss/_search-page.scss]: The toolkit [already has SASS for `.search-result-highlighted-text`](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/search-result.html).

File: app/presenters/search_presenters.py [hunk 2]
```diff
@@ -195,3 +196,19 @@ def __set_filter_states(self):
...
                             filter['value'] in param_values
                         )
+
+
+class SearchResults(object):
+    """Provides access to the search results information"""
+
+    def _add_highlighting(self):
+        for index, service in enumerate(self.search_results):
+            if ('highlight' in service):
```
Comment (quis) [inline app/presenters/search_presenters.py]: This isn't Javascript :wink:

Comment (tombye) [inline app/presenters/search_presenters.py]: Good point, corrected in a47373c

File: app/assets/scss/_search-page.scss (hunk 1 above)
Comment (tombye) [inline app/assets/scss/_search-page.scss]: Yeah, sorry I missed that. I've added some code to make elasticsearch produce HTML that will let us use it: https://github.com/alphagov/digitalmarketplace-search-api/pull/33.

I've added some new SCSS that will use them instead (once that pull request is merged).

**Pull request merged**